src/core/instance.h

```diff
@@ -349,6 +349,12 @@ struct MVMInstance {
     /* Cache the environment hash */
     MVMObject      *env_hash;
 
+    /* Cache the signal hash */
+    MVMObject      *sig_hash;
```

Comment:
I missed this in my last review, but this one needs to be marked in the instance roots function in `src/gc/roots.c`.

Reply:
Thanks. This has been addressed in the latest commits.